cmd/cut-release/main.go

```diff
@@ -115,6 +87,9 @@ func updateHardCodedVersions(version string) {
 	regex = `"version": "(.*)"`
 	updateFileWithRegex(browserPackageJSONPath, regex, newVersionString)
 	newBrowserLiteDependencyString := fmt.Sprintf(`"@0x/mesh-browser-lite": "^%s"`, version)
+	// NOTE(jalextowle): `@0x/mesh-browser` uses the local version of `@0x/mesh-browser-lite`
```

Comment:
@jalextowle `@0x/mesh-browser-lite` has been published now. Do we need to change anything here?

Reply:
Nope. This comment is a bit misleading. Basically, we’re just overwriting the “@0x/mesh-browser-lite” dependency so that it refers to the new version number (which means that @0x/mesh-browser-lite should be published to npm first).